<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pzDialog</class>
 <widget class="QDialog" name="pzDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1576</width>
    <height>845</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>-40</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>41</width>
     <height>41</height>
    </size>
   </property>
   <property name="toolTip">
    <string>Dodaj model</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="zasoby.qrc">
     <normaloff>:/zasoby/add.png</normaloff>:/zasoby/add.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>26</width>
     <height>26</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSzukaj">
   <property name="geometry">
    <rect>
     <x>772</x>
     <y>2</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>41</width>
     <height>41</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="zasoby.qrc">
     <normaloff>:/zasoby/find.png</normaloff>:/zasoby/find.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>26</width>
     <height>26</height>
    </size>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>49</y>
     <width>1561</width>
     <height>793</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="alternatingRowColors">
    <bool>true</bool>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>9</y>
     <width>752</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <item>
     <widget class="QLineEdit" name="lineEditwz">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>124</width>
        <height>31</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditsp">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>124</width>
        <height>31</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditkol">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>124</width>
        <height>31</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditoc">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>124</width>
        <height>31</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditma">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>124</width>
        <height>31</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditwkl">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>124</width>
        <height>31</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Minimum</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>6</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>2</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>41</width>
     <height>41</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="zasoby.qrc">
     <normaloff>:/zasoby/reset.png</normaloff>:/zasoby/reset.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>26</width>
     <height>26</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="zasoby.qrc"/>
 </resources>
 <connections/>
</ui>
